routes/foods.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+const router = express.Router();
+const Food = require('../models/food')
+
+const environment = process.env.NODE_ENV || 'development'
+const configuration = require('../knexfile')[environment]
+const database = require('knex')(configuration)
+
+
+/* GET all foods page. */
+router.get('/', (req, res) => {
+  Food.all()
+    .then((data) => {
+      res.status(201).json(data.rows)
+    })
+});
+
+/* Post a food. */
+router.post('/', (req, res) => {
+  let food_attrs = JSON.parse(req.body.food)
```

Comment:
I was following the 'organize an express app request' lesson and [when they created a variable like this](http://backend.turing.io/module4/lessons/organize-an-express-app), they didn't have to parse the JSON -- does this suggest I'm doing something wrong in how I'm sending my post? @corywest @kat3kasper